sessionctx/variable/session.go

```diff
@@ -93,7 +93,10 @@ type TransactionContext struct {
 	SchemaVersion int64
 	StartTS       uint64
 	Shard         *int64
-	TableDeltaMap map[int64]TableDelta
```

Comment:
Any other attributes of TransactionContext should be moved into Mu?

Reply:
Keep a small lock scope makes better performance.
If we found data race on them, I would update.